include/aeb_opcodes.hrl

```diff
@@ -143,6 +143,11 @@
 -define(         'RETURN', 16#f3).
 -define(   'DELEGATECALL', 16#f4).
 
+-define(        'DBG_LOC', 16#b1).
```

Comment:
This is AEVM, right?

Reply:
No, fate

Reply:
Are you sure? I think `aeb_opcodes.hrl` is only used for AEVM byte code as far as I can see. So I don't think we need to add debug instructions there?

Reply:
@hanssv I've reverted the changed to `aeb_opcodes.hrl` and `aeb_opcodes.erl` here 5eaae614ce496b2c9e3a6da4124fd1dd62da4af3.